packages/@react-spectrum/searchfield/docs/SearchField.mdx

```diff
@@ -0,0 +1,175 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/searchfield';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import Refresh from '@spectrum-icons/workflow/Refresh';
+import {SearchField} from '@react-spectrum/searchfield';
+```
+
+# SearchField
+
+<p>{docs.exports.SearchField.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['SearchField']} />
+
+## Example
+
+```tsx example
+<SearchField
+  label="Search"
+  placeholder="Enter text"
+  onSubmit={text => alert(`Search for ${text}`)} />
+```
+
+## Value
+
+A SearchFields's `value` is empty by default, but an initial, uncontrolled, value can be provided using the `defaultValue` prop.
+Alternatively, a controlled value can be provided using the `value` prop.
+
+```tsx example
+<SearchField defaultValue="Default text" label="Uncontrolled SearchField" marginEnd="20px" />
+<SearchField value="Default text" label="Controlled SearchField" />
+```
+
+Placeholder text that describes the expected value or formatting for the SearchField can be provided using the `placeholder` prop. 
+Placeholder text will only appear when the SearchField is empty, and should not be used as a substitute for labeling the component with a visible label.
+
+```tsx example
+<SearchField placeholder="abc@adobe.com" label="Email" />
+```
+
+## Labeling
+
+A visual label should be provided for the SearchField using the `label` prop. If the SearchField is required, the `isRequired` and 
+`necessityIndicator` props can be used to show a required state.
```

Comment:
I know search field has inherited necessityIndicator, but is that actually something a search field should ever display? Why would a search field ever be required for a form? it doesn't really provide data

Reply:
if the form only has a search field and a submit button, then its basically required, no?

i'm happy to adjust if you think it should be adjusted.

Reply:
Yeah, this one is really more of a question/pondering on my part, haha. Why would a form have a search? or why would a required form have a search? in what scenario would a search be required?
I'll still approve, but now i'm curious, haha